docs/node/run-your-node/prerequisites/stake-requirements.md

```diff
@@ -1,59 +1,66 @@
 # Stake Requirements
 
 This page provides an overview of the stake requirements to become a validator
-on the consensus layer of the Oasis Network.
+on the consensus layer and to run a ParaTime compute node of the Oasis Network.
 
-## Mainnet
+## Consensus Layer
 
-To become a validator on the Oasis Network, you will need to have enough
-tokens staked in your escrow account.
+To become a validator on the consensus layer, you will need to have enough ROSE
+for **registering your entity**, **registering your validator node**, and be among
+the top entities by stake to be included in the **validator set**.
 
-Currently, you are going to need:
+Current requirements are:
 
-* 100 ROSE staked for your entity's registration since that is the
-[staking thresholds].
-* 100 ROSE staked for your validator node's registration since that is the
-[staking thresholds].
-* If you want to be part of the active validator set, you will need enough
-ROSE staked --  this means you will have to be one of the top 120 entities
-by stake. You can check out current top 120 entities on the blockchain explorer
-such as [Oasis Scan].
+|                             | Mainnet  | Testnet  |
+| --------------------------- | -------- | -------- |
+| Entity Registration         | 100 ROSE | 100 TEST |
+| Validator Node Registration | 100 ROSE | 100 TEST |
+| Validator Set Size          | 120      | 110      |
 
-For more information about obtaining information on your entity's account, see
-the [Account Get Info] doc.
+You can check the current top validators on the blockchain explorers like the [Oasis Scan].
 
-Staking thresholds may change in the future. Read the
-[native token information] to see the current values used by the network.
+:::tip
+
+You can use the Oasis CLI to check your account and the network parameters. Use
+the [`oasis account show`] command to check your entity's account and
+[`oasis network show native-token`] to see the current values used by the network.
+
+:::
 
-[staking thresholds]: ../../genesis-doc.md#staking-thresholds
-[Oasis Scan]: https://www.oasisscan.com/validators
-[Account Get Info]: ../../../general/manage-tokens/cli/account.md#show
-[native token information]: ../../../general/manage-tokens/cli/network.md#show-native-token
 
 :::info
 
 To determine if you are eligible to receive a delegation from the Oasis Protocol
 Foundation, see the [Delegation Policy] document.
 
-[Delegation Policy]: ../../../get-involved/delegation-policy.md
-
 :::
 
 :::info
 
 The size of the consensus committee (i.e. the validator set) is configured by
-the [**max_validators** consensus parameter].
+the [**max_validators** consensus parameter][**max_validators** consensus parameter].
+
+:::
 
+For the Testnet, you can receive a limited number of TEST tokens by using our [Oasis Network Testnet Faucet][faucet-testnet].
+For more tokens please contact us on our official [Discord in the #testnet channel][discord].
```

Comment:
I would move this above the admonition boxes.

Reply:
Yes, I was not sure. My thought was the boxes are more important and should be put first, but like this probably no one ever will read the faucet paragraph.